Tools/ardupilotwaf/boards.py

```diff
@@ -528,7 +528,6 @@ def configure_env(self, cfg, env):
             '-Wfatal-errors',
             '-Werror=uninitialized',
             '-Werror=init-self',
-            '-Wframe-larger-than=1024',
```

Comment:
we don't want to remove this globally or we will end up with large stack fns and get stack overflows

Reply:
we already have another flag in this array